tools/clang/unittests/HLSL/ShaderOpTest.cpp

```diff
@@ -1795,6 +1884,32 @@ void ShaderOpParser::ParseDescriptor(IXmlReader *pReader, ShaderOpDescriptor *pD
       CHECK_HR(ReadAttrUINT(pReader, L"WSize", &pDesc->UavDesc.Texture3D.WSize));
       break;
     }
+  } else if (isCBV) {
```

Comment:
Is there nothing to do in this case? or is there more to fill out here? isCBV case.

Reply:
Correct.